massa-models/src/node_configuration/default.rs

```diff
@@ -174,6 +174,8 @@ pub const BLOCK_ID_SIZE_BYTES: usize = massa_hash::HASH_SIZE_BYTES;
 pub const ENDORSEMENT_ID_SIZE_BYTES: usize = massa_hash::HASH_SIZE_BYTES;
 /// operation id size
 pub const OPERATION_ID_SIZE_BYTES: usize = massa_hash::HASH_SIZE_BYTES;
+/// operation id prefix size
+pub const OPERATION_ID_PREFIX_SIZE_BYTES: usize = 20;
```

Comment:
```suggestion
pub const OPERATION_ID_PREFIX_SIZE_BYTES: usize = 10;
```

Reply:
https://github.com/massalabs/massa/issues/2584#issuecomment-1164186662

Reply:
There were more discussions and we settled for 17 :)